mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -2370,6 +2370,26 @@ func ResourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+{{ if ne $.TargetVersionName `ga` -}}
+			"pod_autoscaling": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				Description: `The config for pod autoscaling.`,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"hpa_profile": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: `Horizontal Pod Autoscaling profile. See [HPAProfile](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#hpaprofile) for allowed values.`,
+							DiffSuppressFunc: tpgresource.EmptyOrDefaultStringSuppress("HPA_PROFILE_UNSPECIFIED"),
```

Comment:
Generally we don't expect users to specify UNSPECIFIED as a value; if the enum is expected to change in the future, you could still validate that they aren't setting UNSPECIFIED if it would cause diffs. That would be preferable to a diff suppress func, unless UNSPECIFIED has a special meaning and users need to be able to send it (which is generally not the case.)